cmd/wallet/common.go

```diff
@@ -10,7 +10,8 @@ func isUTXOSpendable(entry *appmessage.UTXOsByAddressesEntry, virtualSelectedPar
 	if !entry.UTXOEntry.IsCoinbase {
 		return true
 	}
-	blockBlueScore := entry.UTXOEntry.BlockBlueScore
+	blockBlueScore := entry.UTXOEntry.BlockDAAScore
+	// TODO: Check for a better alternative than virtualSelectedParentBlueScore
```

Comment:
Do we have an open issue for this?

Reply:
Yes https://github.com/kaspanet/kaspad/issues/1599